src/lib.rs

```diff
@@ -172,15 +180,15 @@ mod tests {
 
         let proof = groth16::prove(&qap, (&sigmag1, &sigmag2), &weights);
 
-        assert!(!groth16::verify(
-            &qap,
+        assert!(!groth16::verify::<CoefficientPoly<FrLocal>, _, _, _, _>(
             (sigmag1, sigmag2),
             &vec![FrLocal::from(2), FrLocal::from(25)],
             proof
         ));
     }
 
-    fn to_bits(mut n: u8) -> [u8; 8] {
+    fn to_bits(num: u8) -> [u8; 8] {
+        let mut n = num;
```

Comment:
Any particular reason for this change? Since `n` is a `u8` it will be copied in so the old version is equivalent to the new one. In my opinion, the original version is a little clearer.

Reply:
I don't remember why I changed that, might have been while I was playing with my own implementation of that logic in circuits. I'll revert it